### PR TITLE
Establish project layout

### PR DESCRIPTION
**Pull request by cimdalli**

### CODE REVIEW

**Comment by cimdalli** [comment]
[(8) Create application layout](https://trello.com/c/kOReoTN4/2-8-create-application-layout)

File: webpack.dev.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
-const merge = require("webpack-merge")
-const common = require("./webpack.common.js")
+const merge = require('webpack-merge')
+const common = require('./webpack.common.js')
```
Comment (omer-demir) [inline webpack.dev.js]: Shouldn't we have output for CD?

**Comment by omer-demir** [comment]
@cimdalli trello issue cant be accessed. I created boards yesterday. can you move to there?

File: webpack.dev.js (hunk 1 above)
Comment (cimdalli) [reply webpack.dev.js]: we have, check appveyor artifacts

**Comment by cimdalli** [comment]
[(8) Create application layout](https://trello.com/c/kOReoTN4/2-8-create-application-layout)

File: webpack.dev.js (hunk 1 above)
Comment (omer-demir) [reply webpack.dev.js]: yep i saw that. thnx

**Comment by omer-demir** [comment]
Can you also update [Readme](./README.md) with `AppVeyor` link

**Comment by cimdalli** [comment]
@omer-demir it's up to date. when you merge to develop and master, builds will be passed.

**Pull request merged**